### ISSUE TITLE
Cannot able to see files in geotrellis-landsat-emr-demo

### ISSUE THREAD

**Issue by rajesh7221** [geotrellis/geotrellis-landsat-emr-demo#26]
upload: viewer/site.tgz to s3://geotrellis-test/emr/site.tgz

upload: server/target/scala-2.10/server-assembly-0.1.0.jar to s3://geotrellis-test/emr/server-assembly-0.1.0.jar

upload: ingest/target/scala-2.10/ingest-assembly-0.1.0.jar to s3://geotrellis-test/emr/ingest-assembly-0.1.0.jar

The three above mentioned files are missing in the repository. Please look in to it if they are available indifferent path.

**Comment by pomadchin** [comment]
Hi @rajesh7221 ! These files should not be in the repo, they can be build from this repo sources. Try to run `make upload-code`.

**Comment by rajesh7221** [comment]
Thanks for the quick response.While running this command 'make upload-code'.Do we need to update the files in config-aws.mk or any other place. Because I cannot able to get the above mentioned files it is ending up with errors. Please let me know

**Comment by pomadchin** [comment]
Yes you need your own credentials file. The reference on how to build this project can be found [here (Makefile ref)](https://github.com/geotrellis/geotrellis-landsat-emr-demo/blob/master/Makefile#L25-L35), and [here (Readme.md ref)](https://github.com/geotrellis/geotrellis-landsat-emr-demo#running-locally).
